alchimia/engine.py

```diff
@@ -41,14 +41,18 @@ def container():
 
 class TwistedEngine(object):
     def __init__(self, pool, dialect, url, reactor=None,
-                 create_worker=_threaded_worker, **kwargs):
+                 create_worker=_threaded_worker,
+                 customize_sub_engine=None,
```

Comment:
A note to the future: we need to talk about what this API looks like before allowing it to be public.

Maybe it should be `_customize_sub_engine`?

Reply:
I can make it a kwarg-only argument for the time being with that name.

Reply:
(But applications will need this functionality if they want to use sqlite...)